templates/template/README.md

```diff
@@ -79,24 +78,103 @@ Before submitting any PRs, a Github Workflow will check to see if your package d
 
 {{- end }}
 
+{{- if (eq .Template "source") }}
+### Making Changes
+
+As a developer making changes to a particular package, you will usually follow the following steps:
+
+#### If this is the first time you are adding a package:
+
+```shell
+PACKAGE=<packageName>
+mkdir -p packages/${PACKAGE}
+touch packages/${PACKAGE}/package.yaml
+```
+
+See `packages/README.md` to configure the `packages/${PACKAGE}/package.yaml` file based on the Package that you are planning to add. To make changes, see the steps listed below.
+
+#### If the package already exists
+
+If you are working with a single Package, set `export PACKAGE=<packageName>` to inform the scripts that you only want to make changes to a particular package. This will prevent the scripts from running commands on every package in this repository.
+
+You'll also want to update the `packageVersion` and `releaseCandidateVersion` located in `packages/${PACKAGE}/package.yaml`. See the section below for how to update this field.
+
+Once you have made those changes, the Workflow will be:
+```shell
+make prepare # Instantiates the chart in the workingDir specified in the package.yaml
+# Make your changes here to the workingDir directly here
+make patch # Saves changes to generated-changes/
+make clean # Cleans up your workingDir, leaving behind only the generated-changes/
+```
+
+Once your directory is clean, you are ready to submit a PR.
+
+#### Versioning Packages
+
+If this major.minor.patch version of the Chart has never been released, reset the `packageVersion` to `01` and the `releaseCandidateVersion` to `00`.
+
+If this major.minor.patch version of the Chart has been released before:
```

Comment:
nit: wrap major.minor.patch with "\`"  characters like so `major.minor.patch`

Reply:
addressed in latest, thanks for catching!